### PR TITLE
Deduplicate PMC/Committer addition stuff. 

### PR DESCRIPTION
**Pull request by rbowen**
Deduplicates new committer and new pmc member documentation. Redirects old URLs. Updates various links to and from these documents to compensate for new location.
The New Committer doc is a straight move. The New PMC Member doc consolidates content from two versions of this doc, and so could probably use some additional editing or wordsmithing.
Resolves COMDEV-551, COMDEV-552, and COMDEV-555

### CODE REVIEW

**Review by Kingveal — APPROVED** [review]

File: source/pmc/adding-committers.md
Comment (ShaneCurcuru) [inline source/pmc/adding-committers.md:29]: Suggest making link text detailed, not generic (which sounds like "click here" to me somehow).  So:

Use the [discussion template for a new committer]...

File: source/pmc/adding-committers.md
Comment (ShaneCurcuru) [inline source/pmc/adding-committers.md:252]: Like this - using consistent link text throughout will help reinforce what specific links/documents mean.

**Review by ShaneCurcuru — CHANGES_REQUESTED** [review]
Yes! Much more readable and useful! LGTM, modulo the link texts being descriptive.

File: source/pmc/adding-committers.md
Comment (rbowen) [reply source/pmc/adding-committers.md:254]: Done.

**Review by Kingveal — APPROVED** [review]

**Comment by Kingveal** [comment]
not bad lol im only 23

**Pull request merged**
